src/ucp/rndv/rndv_ppln.c

```diff
@@ -174,21 +174,22 @@ ucp_proto_rndv_ppln_frag_complete(ucp_request_t *freq, int send_ack,
 
 void ucp_proto_rndv_ppln_send_frag_complete(ucp_request_t *freq, int send_ack)
 {
-    ucp_proto_rndv_ppln_frag_complete(freq, send_ack,
+    ucp_proto_rndv_ppln_frag_complete(freq, send_ack, 0,
                                       ucp_proto_request_complete_success,
                                       "ppln_send");
 }
 
 void ucp_proto_rndv_ppln_recv_frag_clean(ucp_request_t *freq)
 {
     ucp_proto_rndv_ppln_frag_complete(
-            freq, 0, (ucp_proto_complete_cb_t)ucs_empty_function,
+            freq, 0, 0, (ucp_proto_complete_cb_t)ucs_empty_function,
             "ppln_recv_clean");
 }
 
 void ucp_proto_rndv_ppln_recv_frag_complete(ucp_request_t *freq, int send_ack)
 {
     ucp_proto_rndv_ppln_frag_complete(freq, send_ack,
+                                      UCS_STATUS_IS_ERR(freq->status),
```

Comment:
I'm not sure if it would speed up somehow noticeable not inlined function, but OK it makes sense from perspective of design and existing code alignment.
BTW, CI failures happened because of `status` was not initialized. Is it intentionally or by mistake?